src/HolographicCamera.Unity/Assets/HolographicCamera/Scripts/KeepDeviceActive.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+#if UNITY_WSA && WINDOWS_UWP
+    using System;
+    using Windows.System.Display;
+#endif
+
+public class KeepDeviceActive : MonoBehaviour
+{
+#if UNITY_WSA && WINDOWS_UWP
+    private static DisplayRequest displayRequest;
+#endif
+
+    private void Start()
+    {
+        CreateDisplayRequest();
+    }
+
+    private void OnDestroy()
+    {
+#if UNITY_WSA && WINDOWS_UWP
+        if (displayRequest != null)
```

Comment:
>f (displayRequest != null) [](start = 9, length = 26)

Does this need the InvokeOnUIThread wrapper as well?

Reply:
Yeah, seems like it should